app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,67 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all.order(:completed, :priority, :created_at)
+  end
+
+  def show
+    @task = Task.find_by(id: params[:id])
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    task = Task.new(task_params)
+    if task.save
```

Comment:
You either need to have a hidden field on completion status (set to false) in the form, or set:
`task.completed = false`
before saving